### PR TITLE
Tutorial

### PR DESCRIPTION
**Pull request by RohitRathore1**

### CODE REVIEW

File: docs/src/assets/german_credit.csv [hunk 1]
```diff
@@ -0,0 +1,1001 @@
+,Age,Sex,Job,Housing,Saving accounts,Checking account,Credit amount,Duration,Purpose
```
Comment (bkamins) [inline docs/src/assets/german_credit.csv:1]: why rowid column does not have a name?

File: docs/src/assets/readme.txt [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Source
```
Comment (bkamins) [inline docs/src/assets/readme.txt:1]: if you use markdown then better name it readme.md (this is even better)

File: docs/src/assets/german_credit.csv (hunk 1 above)
Comment (RohitRathore1) [reply docs/src/assets/german_credit.csv:1]: Oh, yeah. Fixing it. In the kaggle they had mentioned using #.

File: docs/src/assets/readme.txt (hunk 2 above)
Comment (RohitRathore1) [reply docs/src/assets/readme.txt:1]: Yeah, fixing it.

Comment (bkamins) [reply docs/src/assets/readme.txt:1]: I think it is enough to do a list of files we provide in assets. And for each file add information about the original source, the source from which we have taken the file and a link to the license we use (like Kaggle - they just give a link to the license)

File: docs/src/assets/readme1.txt [hunk 3]
```diff
@@ -0,0 +1,487 @@
+# Source
```
Comment (bkamins) [inline docs/src/assets/readme1.txt:1]: as per comment above then one file would be enough

File: docs/src/assets/readme.md [hunk 4]
```diff
@@ -0,0 +1,439 @@
+# Source
```
Comment (bkamins) [inline docs/src/assets/readme.md:1]: As commented - I would avoid adding this much detail.

Just write something like

```
# Introduction

In this folder we store the following data sets:

* german_credit.csv
* iris.csv

# German Credit data set

## License:
link

## Source:

both info that the original data is from UCI, and that this data is from Kaggle (as this information is missing)
```

If you feel uncomfortable with something just leave an empty space and I will fill it

Comment (RohitRathore1) [reply docs/src/assets/readme.md:1]: Oh okay, i had not read your above comment.

File: docs/src/assets/readme.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+Professor Dr. Hans Hofmann
+Institut f"ur Statistik und "Okonometrie
+Universit"at Hamburg
+FB Wirtschaftswissenschaften
+Von-Melle-Park 5
+2000 Hamburg 13
+
+The original data is from [UCI](https://archive.ics.uci.edu/ml/datasets/statlog+(german+credit+data)), and that this data is from [Kaggle](https://www.kaggle.com/uciml/german-credit)
+
+
```
Comment (bkamins) [inline docs/src/assets/readme.md:24]:
```suggestion
# Iris data set

## License
https://creativecommons.org/publicdomain/zero/1.0/

## Source:
https://archive.ics.uci.edu/ml/datasets/Iris
Creator:
R.A. Fisher
```

**Comment by bkamins** [comment]
Given you have opened https://github.com/JuliaData/DataFrames.jl/pull/2700 please make sure to correctly handle rebasing of this PR before submitting it for a review (as I assume in this PR you will add tutorial sections and in https://github.com/JuliaData/DataFrames.jl/pull/2700 you propose to change the legacy structure of the tutorial).

File: docs/src/assets/readme.md (hunk 5 above)
Comment (bkamins) [reply docs/src/assets/readme.md:25]: in the future you can just clik "Commit suggestion" (or edit it before accepting and then commit)

**Comment by bkamins** [comment]
Here I meant rebasing because I assumed that most likely this PR would have conflicts with other changes you make. Therefore it is better to open it again against latest `main`. I assume this is what you are planning to do. Right?

**Pull request closed without merging**